mantidproject/mantid
# Removed unused documentation images

Description of work.
This PR removes some unused documentation images. 
Here is a [script](https://gist.github.com/DavidFair/70cc14348dc02ab6f42aedb9be13dfdb) for future use that tests whether each image file in a folder is used, and deletes it if it isn't

**To test:**
Run `docs-test` target locally.
Ensure there are no warnings or errors in the build.

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
